lib/smart_proxy_pulp_plugin/pulpcore_plugin.rb

```diff
@@ -5,19 +5,16 @@
 module PulpProxy
   class PulpcorePlugin < ::Proxy::Plugin
     plugin "pulpcore", ::PulpProxy::VERSION
-    default_settings :pulp_url => 'https://localhost',
-                     :content_app_url => 'https://localhost:24816/',
-                     :mirror => false,
-                     :client_authentication => ['password', 'client_certificate'],
-                     :rhsm_url => 'https://localhost/rhsm'
+    default_settings :mirror => false,
+                     :client_authentication => ['password', 'client_certificate']
 
     AUTH_TYPES = ['password', 'client_certificate'].freeze
 
     load_validators include: ::PulpProxy::Validators::Include
-    validate :pulp_url, :url => true
-    validate :content_app_url, :url => true
+    validate :pulp_url, :presence => true, :url => true
```

Comment:
not sure `presence` is really needed -- `url` already checks it's a valid URL which `nil` isn't?